morphemes.json

```diff
@@ -7750,16 +7750,7 @@
         "origin"        : "Latin",
         "isPrefix"      : false,
         "isSuffix"      : false,
-        "notes"         : "scale as in the scales of a reptile."
```

Comment:
this diff makes more sense if you look at the full context - this entry is a dupe